contracts/container/contract.go

```diff
@@ -525,6 +560,107 @@ func Create(cnr []byte, invocScript, verifScript, sessionToken []byte, name, zon
 	runtime.Notify("Created", id, owner)
 }
 
+// CreateV2 creates container with given info in the contract. Created
+// containers are content-addressed: they may be accessed by SHA-256 checksum of
+// their data. On success, CreateV2 throws 'Created' notification event.
+//
+// Created containers are disposable: if they are deleted, they cannot be
+// created again. CreateV2 throws [cst.ErrorDeleted] exception on recreation
+// attempts.
+//
+// If '__NEOFS__NAME' and '__NEOFS__ZONE' attributes are set, they are used to
+// register 'name.zone' domain for given container. Domain zone is optional: it
+// defaults to the 6th contract deployment parameter which itself defaults to
+// 'container'.
+//
+// If '__NEOFS__METAINFO_CONSISTENCY' attribute is set, meta information about
+// objects from this container can be collected for it.
+//
+// The operation is paid. Container owner pays per-container fee (global chain
+// configuration) to each committee member. If domain registration is requested,
+// additional alias fee (also a configuration) is added to each payment.
+//
+// CreateV2 must have chain's committee multi-signature witness. Invocation
+// script, verification script and session token parameters are owner
+// credentials. They are transmitted in notary transactions carrying original
+// users' requests. IR verifies requests and approves them via multi-signature.
+// Once creation is approved, container is persisted and becomes accessible.
+// Credentials are disposable and do not persist in the chain.
+func CreateV2(cnr Info, invocScript, verifScript, sessionToken []byte) {
```

Comment:
How about returning a Hash256 from it? Currently it'd be known, but we can use more interesting schemes in future. Although the hash is known from the event also.